hazelcast/src/main/java/com/hazelcast/map/impl/operation/PartitionWideEntryOperation.java

```diff
@@ -76,12 +76,18 @@ public void run() {
         responses = new MapEntries(recordStore.size());
         operator = operator(this, entryProcessor, getPredicate(), true);
 
-        Iterator<Record> iterator = recordStore.iterator(Clock.currentTimeMillis(), false);
+        Iterator<Record> iterator = recordStore.loadAwareIterator(Clock.currentTimeMillis(), true, false);
```

Comment:
The javadoc of the class needs to be updated. 
If I understand correctly, before the record store was checked if it was loaded and entry access stats were updated on each call to `DefaultRecordStore#get` and now we moved this logic to the iterator. 
Is this correct? Why did it not load the entries before? What do we achieve with this?